tests/pyreverse/test_main.py

```diff
@@ -0,0 +1,25 @@
+import os
+import sys
+
+import pytest
+
+from pylint.lint import fix_import_path
+
+TEST_DATA_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "data"))
+PROJECT_ROOT_DIR = os.path.abspath(os.path.join(TEST_DATA_DIR, ".."))
+
+
+@pytest.fixture(name="setup_path", params=[PROJECT_ROOT_DIR, TEST_DATA_DIR])
+def fixture_setup_path(request):
```

Comment:
```suggestion
def fixture_setup_path(request: ?) -> Iterator[str]:
```
Could you add typing, please ? :)

Reply:
Sure.